packages/core/src/components/flip-tile/test/flip-tile.ct.ts

```diff
@@ -59,3 +59,24 @@ test('should change content', async ({ mount, page }) => {
   await expect(flipContentOneElement).not.toBeVisible();
   await expect(flipContentTwoElement).toBeVisible();
 });
+
+test('toggle - prevent default', async ({ mount, page }) => {
+  await mount(`
+    <ix-flip-tile>
+      <ix-flip-tile-content>Page 1</ix-flip-tile-content>
+      <ix-flip-tile-content>Page 2</ix-flip-tile-content>
+    </ix-flip-tile`);
+  const flipTile = page.locator('ix-flip-tile');
+  await expect(flipTile).toHaveClass(/hydrated/);
+  await flipTile.evaluate((d: HTMLIxFlipTileElement) =>
+    d.addEventListener('toggle', (event) => {
+      event.preventDefault();
+    })
+  );
+  await flipTile.locator('ix-icon-button').click();
+  await flipTile.evaluate((d: HTMLIxFlipTileElement) => (d.index = 1));
+  const pageOne = flipTile.locator('ix-flip-tile-content').nth(0);
+  const pageTwo = flipTile.locator('ix-flip-tile-content').nth(1);
+  await expect(pageOne).toBeVisible();
+  await expect(pageTwo).not.toBeVisible();
+});
```

Comment:
You test `index` prop in line 77. We could also add a additional test to test if the index really change the flip content